Src/Base/AMReX_RungeKutta.H

```diff
@@ -0,0 +1,287 @@
+#ifndef AMREX_RUNGE_KUTTA_H_
+#define AMREX_RUNGE_KUTTA_H_
+#include <AMReX_Config.H>
+
+#include <AMReX_FabArray.H>
+
+namespace amrex {
+
+/**
+ * \brief Functions for Runge-Kutta methods
+ *
+ * This namespace RungeKutta has functions for a number RK methods, RK2, RK3
+ * and RK4.  The function templates take the old data in FabArray/MultiFab
```

Comment:
Can you include some context/naming here (or butcher tables later?) For instance, is RK3 a specific SSPRK3 method?

Reply:
Good idea.  I will include links to wikipedia.

By the way, the name SSPRK3 appeared many years after the method was presented in a paper by Shu & Osher.  So I never got used to the name.